### PR TITLE
fix: TITLE 2 instead of proper title in events tab

### LINKED ISSUE #199: TITLE 2 instead of proper title in events tab
### Describe the bug

Events tab has placeholder text "TITLE 2" instead of proper title.

### To Reproduce
1. Go to https://anitab-org.github.io/
2. Click on events
3. See there is `TITLE 2`

### Expected behavior
There should be a proper title

### Screenshots
Screenshot 2021-01-25 at 20 24 40

### Desktop (please complete the following information):
 - OS: any
 - Browser any
 - Version any

### Smartphone (please complete the following information):
 - Device: any
 - OS: any
 - Browser: any
 - Version: any

### PR DESCRIPTION
**Pull request by Amulya-coder**
### Description

Removed TITLE2 and duplicate image as discussed with the mentor

Fixes #199 

### Type of Change:

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
image

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Amulya-coder** [comment]
@annabauza changes made. Please review it.

File: src/content/our_events.js [hunk 1]
```diff
@@ -12,9 +12,8 @@ const our_events={
             ]
         },
         {
-            title: "TITLE 2",
             image:{
-                source:require("./../assets/our_events.png"),
+                
             },
             content: [
```
Comment (annabauza) [inline src/content/our_events.js:18]: Delete lines 12 to 18 inclusive, and we will be where we should be. Thanks!

**Review by annabauza — CHANGES_REQUESTED** [review]
As comment.

File: src/content/our_events.js [hunk 2]
```diff
@@ -12,9 +12,8 @@ const our_events={
             ]
```
Comment (annabauza) [inline src/content/our_events.js:12]:
```suggestion
```

File: src/content/our_events.js [hunk 3]
```diff
@@ -12,9 +12,8 @@ const our_events={
...
         },
```
Comment (annabauza) [inline src/content/our_events.js:13]:
```suggestion
```

File: src/content/our_events.js [hunk 4]
```diff
@@ -12,9 +12,8 @@ const our_events={
...
         {
```
Comment (annabauza) [inline src/content/our_events.js:14]:
```suggestion
```

File: src/content/our_events.js [hunk 5]
```diff
@@ -12,9 +12,8 @@ const our_events={
...
-            title: "TITLE 2",
             image:{
```
Comment (annabauza) [inline src/content/our_events.js:15]:
```suggestion
```

File: src/content/our_events.js [hunk 6]
```diff
@@ -12,9 +12,8 @@ const our_events={
...
-                source:require("./../assets/our_events.png"),
+                
             },
```
Comment (annabauza) [inline src/content/our_events.js:17]:
```suggestion
```

File: src/content/our_events.js (hunk 1 above)
Comment (annabauza) [inline src/content/our_events.js:18]:
```suggestion
```

**Review by annabauza — CHANGES_REQUESTED** [review]

File: src/content/our_events.js [hunk 7]
```diff
@@ -9,14 +9,7 @@ const our_events={
                 {
                     par : "We at AnitaB.org Open Source believe in Inspiration through Action. To facilitate our Open Source journey and continue the AnitaB.org legacy we conduct enthralling events all year long, with the purpose of bridging the gap between creative developers and our Open Source community. Keep scrolling to catch a glimpse of few such events! ",
                 }
```
Comment (annabauza) [inline src/content/our_events.js:11]:
```suggestion
                },
```

Comment (annabauza) [inline src/content/our_events.js]:
```suggestion
                },
```

**Review by annabauza — APPROVED** [review]

**Comment by annabauza** [comment]
I have made required changes. Thanks.

**Pull request merged**